content/doc/pipeline/tour/deployment.adoc

```diff
@@ -143,7 +143,7 @@ node {
 
 This Guided Tour introduced you to the basics of using Jenkins and Jenkins
 Pipeline. Because Jenkins is extremely extensible, it can be modified and
-and configured to handle practically _any_ aspect of automation. To learn more
```

Comment:
Actually, I mistakenly added another 'and' in my previous commit, which meant the documentation would've read '...modified and and configured...'.
I guess I'm still getting used to editing line-wrapped documentation housed in Git. Never used to do this before but it's a good practice - especially for reviews like these.